flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
I can't test arm support, hence I dont really want to included a build that might not work

Reply:
I'll remove this then